sig-cli/CONTRIBUTING.md

```diff
@@ -294,7 +296,7 @@ See the sig-cli [community page] for points of contact and meeting times:
 [feature request]: #feature-requests
 [feature]: https://github.com/kubernetes/features
 [group]: https://groups.google.com/forum/#!forum/kubernetes-sig-cli
-[issue]: https://github.com/kubernetes/kubernetes/issues
+[issue]: https://github.com/kubernetes/kubectl/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20no%3Aassignee%20label%3Afor-new-contributors
 [cli_help_wanted_issues]: https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Asig%2Fcli+label%3Ahelp-wanted
```

Comment:
cli help wanted issues will be living in 2 places: core repo and kubectl repo.
There are only 2 help-wanted issues in kubectl repo atm.
But we may want to update this now or sometime later.